src/corehost/cli/deps_format.cpp

```diff
@@ -68,7 +63,7 @@ void deps_json_t::reconcile_libraries_with_targets(
                 size_t pos = library.first.find(_X("/"));
                 entry.library_name = library.first.substr(0, pos);
                 entry.library_version = library.first.substr(pos + 1);
-                entry.library_type = _X("package");
+                entry.library_type = pal::to_lower(library.second.at(_X("type")).as_string());
```

Comment:
> We can directly go to app base

This should be done by design, implying that project assemblies should figure out in app.deps.json, similar to explicitly pulled in nuget dependencies and thus, from activation standpoint, there is no difference in resolving these vs explicit dependencies.